pkg/apis/serving/v1beta1/predictor_sklearn.go

```diff
@@ -53,22 +52,60 @@ func (k *SKLearnSpec) Default(config *InferenceServicesConfig) {
 
 // GetContainer transforms the resource into a container spec
 func (k *SKLearnSpec) GetContainer(metadata metav1.ObjectMeta, extensions *ComponentExtensionSpec, config *InferenceServicesConfig) *v1.Container {
-	arguments := []string{
-		fmt.Sprintf("%s=%s", constants.ArgumentModelName, metadata.Name),
-		fmt.Sprintf("%s=%s", constants.ArgumentModelDir, constants.DefaultModelLocalMountPath),
-		fmt.Sprintf("%s=%s", constants.ArgumentHttpPort, constants.InferenceServiceDefaultHttpPort),
-	}
-	if extensions.ContainerConcurrency != nil {
-		arguments = append(arguments, fmt.Sprintf("%s=%s", constants.ArgumentWorkers, strconv.FormatInt(*extensions.ContainerConcurrency, 10)))
-	}
+	k.Container.Env = append(
```

Comment:
Does MLServer support command arguments ? If we can use the same set of the arguments that helps maintain the backwards compatibility.